plone/app/contentlisting/realobject.py

```diff
@@ -84,7 +84,9 @@ def getSize(self):
     def review_state(self):
         obj = self.getObject()
         wftool = getToolByName(obj, 'portal_workflow')
-        return wftool.getInfoFor(obj, 'review_state')
+        return wftool.getInfoFor(obj, 'review_state', default=None)
+        # NOTE: Potential candidat for plone.api replacement.
+        # return api.content.get_state(obj=obj, default=None)
```

Comment:
Unless we have no defined list of producers for and consumers of plone.api and unless this package is not defined as a consumer of plone.api, we shall not place such statements in the code. This might get confusing for other developers stumbling over it.